relay-server/src/utils/rate_limits.rs

```diff
@@ -142,6 +142,82 @@ fn infer_event_category(item: &Item) -> Option<DataCategory> {
     }
 }
 
+/// Quantity metrics for a single category of attachments.
+///
+/// Tracks both the count of attachments and size in bytes.
+#[derive(Clone, Copy, Debug, Default)]
+pub struct AttachmentQuantities {
+    count: usize,
+    bytes: usize,
```

Comment:
```suggestion
    pub count: usize,
    pub bytes: usize,
```

Also would add some docs then, it's kinda obvious but a small docstring